src/theme/light-theme.tsx

```diff
@@ -55,84 +62,79 @@ import {
 
 const shadows = [
   'none',
-  ...new Array(24)
-    .fill(0)
-    .map((_, i) => i + 1)
-    .map((i) => [0, i / 12, i, i / 4])
-    .map((v) => v.map(Math.ceil))
-    .map(
-      (v) =>
-        `${v[0]}px ${v[1]}px ${v[2]}px ${v[3]}px ${alpha(sb_dark_blue, 0.07)}`,
-    ),
+  '0px 4px 8px 0px rgba(27, 36, 63, 0.08)',
+  '0px 8px 24px 0px rgba(27, 36, 63, 0.08)',
+  '0px 16px 32px 0px rgba(27, 36, 63, 0.08)',
+  '0px 16px 48px 0px rgba(27, 36, 63, 0.08)',
+  ...Array(20).fill('none'),
 ] as Theme['shadows']
 
 const spacing = (factor?: number) => Math.floor(factor ?? 1) * 4
 
 const palette = {
   mode: 'light',
   primary: {
-    main: color_primary,
-    contrastText: white,
+    main: sb_primary,
+    contrastText: sb_neutral_white,
```

Comment:
What is the design token's definition of "neutral white" is it for text or surfaces? "neutral black" doesn't seem to be intended for text, for example.

Reply:
there there's no definition, it's only a primitive colors for black and white

Reply:
I went back with the way it was https://github.com/storyblok/mui/pull/48/commits/e66a71cf031abebc0d22ee84551b2357dd7379ef#diff-f2727474dc5684e74bd35e134abe55e2a53454471a4f806c72942be9b1b0b458R76